_source/_posts/2021-12-20-holiday-2021.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: blog_post
+title: "Happy Holidays!"
+author: edunham
+by: advocate
+communities: 
+description: "Okta's Developer Advocacy team reflects on what we're grateful for in 2021"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: happy-holidays-okta.jpg
+type: awareness
+---
+
+As the holiday season gets into full swing, the Developer Advocacy team at Okta has sat down around our virtual table to chat about things we're grateful for across our various technical communities. 
+
+{% img avatar-deepu-sasidharan.jpg alt:"Deepu" width:"200" %}
+
+Deepu:
+
+I'm so thankful for Linux accepting Rust as another language for the Kernel and I'm thankful for all the backing it is getting from big names. I'm sure this will lead to Linux becoming more secure as more and more code might be written in Rust.
```

Comment:
```suggestion
I'm so thankful for Linux accepting Rust as another language for the Kernel and I'm thankful for all the backing it is getting from big names. I'm sure this will lead to Linux becoming more secure as more and more code might be written in Rust.

I'm also grateful for great communities like [Foojay](https://foojay.io/) and the awesome contributors of JHipster and Rust
```

Reply:
I thought i'll add a line more :)